test/configs/zombieTanssi.json

```diff
@@ -46,6 +61,7 @@
                 },
                 {
                     "name": "Collator2000-01",
+                    "ws_port": "9960",
```

Comment:
we dont need these

Reply:
Yes, I changed this for local testing only. I'll remove it.